roles/replace_node/tasks/main.yml

```diff
@@ -0,0 +1,62 @@
+---
+# Currently we support only same node replacement.
+# Change remove this when we enhance the role
+- name: Check if old and new node are same
+  fail:
+    msg: "Hostnames of old and new nodes are not same"
```

Comment:
Should we update the README that this version only supports this?

Reply:
Ack! Done.